tests/test_clients.py

```diff
@@ -51,7 +51,7 @@ def test_timing_with_tag(self):
 
     @patch('statsd.client.StatsClient.timer')
     def test_timer_no_tag(self, mocked_super):
-        statsdecor.timer('a.metric')
+        assert statsdecor.timer('a.metric') is not None
```

Comment:
Is there a way to be more specific on what is returned? Do consumers of this library rely on the return value being a specific type?